### DISCUSSION TITLE
how to make a publii cms url like a blogspot url?

### DISCUSSION THREAD

**Discussion by natamisibuea** [GetPublii/Publii#1495] [General]
Hello Publii Team, permission to ask how to set the publii url to be like the blogspot url follow a pattern like yourblogname.blogspot.com/yyyy/mm/your-post-title.html
Thank you.

**Comment by candidexmedia** [comment]
Hi @natamisibuea : Unfortunately, it's not currently possible to add directories in URLs. [Here's how Publii generates URLs](https://getpublii.com/dev/how-urls-in-publii-works/).

See related discussions:

- https://github.com/GetPublii/Publii/discussions/1450#discussioncomment-8875920
- https://github.com/GetPublii/Publii/discussions/1413#discussioncomment-8696773
- https://github.com/GetPublii/Publii/discussions/1466
- https://forum.getpublii.com/topic/folder-structure/
- https://forum.getpublii.com/topic/is-there-a-way-to-create-posts-automatically-inside-subfolders/
- https://forum.getpublii.com/topic/custom-url-path-for-posts/

**Comment by rauschma** [comment]
This is currently the only thing that prevents me from using Publii! I’m willing to pay money for this feature, to help with its funding.

One possible solution: Specify the whole path (instead of only the slug) of a post – e.g.: `2024/07/my-post.html`
